Kernel/VirtIO/VirtIOConsole.cpp

```diff
@@ -62,14 +77,23 @@ bool VirtIOConsole::handle_device_config_change()
 
 void VirtIOConsole::handle_queue_update(u16 queue_index)
 {
+    dbgln_if(VIRTIO_DEBUG, "VirtIOConsole: Handle queue update");
     VERIFY(queue_index <= TRANSMITQ);
     switch (queue_index) {
-    case RECEIVEQ:
+    case RECEIVEQ: {
+        ScopedSpinLock lock{get_queue(RECEIVEQ).lock()};
```

Comment:
```suggestion
        ScopedSpinLock lock(get_queue(RECEIVEQ).lock());
```
To follow the style of the rest of the project

Reply:
Yeah, I just realised that this code needs to be chained, we probably shouldn't be trying to grab a spinlock in an interrupt handler.

Reply:
I talked to @tomuta , apparently it's fine to use a spinlock in an interrupt handler, since we don't allow interrupts while trying to grab one.